pkg/logging/json_logger.go

```diff
@@ -88,10 +88,9 @@ func (jsonLogger *JSONLogger) PreProcess(dataStore string, config *logging.Confi
 	l := &logrotate.Logger{
 		Filename: jsonFilePath,
 	}
-	//maxSize Defaults to unlimited.
-	var capVal int64
-	capVal = -1
+	//maxSize Defaults to 100MB.
```

Comment:
Where is this set to 100MiB?

How was this value chosen?

Reply:
100MB is the default value that go-logrotate specifies for each log file. 
[defaultMaxsize  = 100](https://github.com/fahedouch/go-logrotate/blob/6a8beddaea39b2b9c77109d7fa2fe92053c063e5/logrotate.go#L500)

Reply:
Could you add that context to the code comment?